doc/configuration.rst

```diff
@@ -34,6 +32,20 @@ it is processed. It supports 3 kind of sections:
 - a section for each command
 - a section for each plugin category
 
+.. note::
```

Comment:
We are adding first-class support to TOML... so it deserves its own section on docs. Not just a "note".

I would rather "duplicate" the docs than force users to read one and figure out how to translate it...

Also ok to put TOML first and say it is the preferred format (because it might include extra features).